skimage/graph/_mcp.pyx

```diff
@@ -391,9 +391,13 @@ cdef class MCP:
         """
         pass
 
-
+    @deprecate_parameter(
+        "max_cumulative_cost",
+        start_version="0.25",
+        stop_version="0.27",
+    )
     def find_costs(self, starts, ends=None, find_all_ends=True,
-                   max_coverage=1.0, max_cumulative_cost=None, max_cost=None):
+                   max_coverage=1.0, max_cumulative_cost=DEPRECATED, max_cost=None):
```

Comment:
I love that you took the time to figure out, how we do deprecations. :blush: Only thing missing is a reminder in our `TODO.txt`.

Reply:
Added a comment to the `TODO.txt`